### PR TITLE
Feat/count of games by season

### LINKED ISSUE #30: Create #count_of_games_by_season method and tests
- Returns a hash
- Likely accessed from Games class.

### PR DESCRIPTION
**Pull request by wallacebilly1**
Close #30

### CODE REVIEW

File: spec/game_spec.rb [hunk 1]
```diff
@@ -1,57 +1,64 @@
...
-      season: 20122013,
-      type: "Postseason",
-      away_team_id: 3,
-      home_team_id: 6,
-      away_goals: 2,
-      home_goals: 3
-    }
-    @game1 = Game.new(game_data)
-
+  before(:all) do
```
Comment (AuroraTuring) [inline spec/game_spec.rb:4]: Nice use of before(:all), thanks for showing us this!

**Comment by AuroraTuring** [comment]
Everything looks good to me!

**Review by grantdavis303 — APPROVED** [review]
Looks good Billy!

**Review by AuroraTuring — APPROVED** [review]
Grant says everything passes, we're good to go!

**Pull request merged**